lib/Settings/BeforePreferenceSetEventListener.php

```diff
@@ -76,6 +76,11 @@ public function validatePreference(string $userId, string $key, string|int|null
 			return $valid;
 		}
 
+		// "compact-list" 0/1
+		if ($key === UserPreference::CONVERSATIONS_COMPACT_LIST) {
+			return is_numeric($value);
```

Comment:
Need to confirm it's only 0 or 1 for now (see line 71)
But instead I would change to speaking strings instead of boolean integers